src/content/docs/telemetry-data-platform/manage-data/query-limits.mdx

```diff
@@ -204,5 +204,55 @@ Attributes on `newrelic.resourceConsumption.impact`
 - `Impact` - A count of what is happening when resource has exceeded set limit, i.e dropped requests. 
 - `consumingAccountId` - The New Relic account where the resource is being consumed.
 
- 
+## Set alerts on resource metrics
+
+While building a dashboard to see all your limits is handy, being able to automate it is even better. You can set alerts on your limit metrics to provide updates on limits changes.
```

Comment:
I have decent idea but regarding 'resource metrics' and 'limit metrics'; maybe we should use same wording for these or else explain at high level what we're talking about (metrics related to your usage) and maybe also clarifying that these can be used by querying the `Metrics` data type.